recurring_ical_events.py

```diff
@@ -130,13 +130,14 @@ def __init__(self, source, start, stop, keep_recurrence_attributes=False):
         self.source = source
         self.start = start
         self.stop = stop
+        self.end_prop = end_prop
         self.keep_recurrence_attributes = keep_recurrence_attributes
 
     def as_vevent(self):
```

Comment:
My guess is that this will be part of the subclass, too..

Reply:
There is one single place relatively deep in a method in the super class that uses this one.  I didn't see any obvious way to move that logic to the subclasses without adding code duplication.  You're welcome to have a look though :-)